src/status.js

```diff
@@ -53,6 +37,74 @@ const ShowList = ((arr) => {
   });
 });
 
+const addToList = (() => {
+  const toDoTask = {
+    id: collection.length,
+    description: document.getElementById('todotitle').value,
+    completed: false,
+  };
+  collection.push(toDoTask);
+  localStorage.setItem('todoList', JSON.stringify(collection));
+  if (collection.length > 0) {
+    ShowList(collection);
+  }
+  form.reset();
+});
+
+const editTask = ((ev) => {
+  const buttonID = ev.target.id;
+  const dataGet = localStorage.getItem('todoList');
+  const data = JSON.parse(dataGet);
+  if (data) {
+    collection = data;
+  }
+  const btnId = collection[collection.findIndex((x) => x.id === parseInt(buttonID, 10))];
+  const last = collection.indexOf(btnId);
+  const test = ev.target.parentNode;
+  const editInput = test.querySelector('p');
+  editInput.contentEditable = true;
+  editInput.classList.add('test');
+  const a = test.querySelector('.fa-ellipsis-v');
+  a.style.display = 'none';
+  const b = test.querySelector('.fa-trash-o');
+  b.style.display = 'block';
+  collection[last].description = editInput.innerHTML;
+
+  editInput.addEventListener('keyup', () => {
+    if (editInput.innerHTML.length > 0) {
+      const listItem = document.getElementById('showListItem');
+      const errorMsg = listItem.querySelector('small');
+      errorMsg.style.display = 'none';
+      collection[last].description = editInput.innerHTML;
+      localStorage.setItem('todoList', JSON.stringify(collection));
+    } else if (editInput.innerHTML.length === 0) {
+      const listItem = document.getElementById('showListItem');
+      const errorMsg = listItem.querySelector('small');
+      errorMsg.style.display = 'block';
+      errorMsg.innerText = 'Please enter some text';
+      errorMsg.classList.add('small');
+    }
+  });
+});
+
+const removeTodo = ((ev) => {
+  const buttonId = ev.target.id;
+  collection = collection.filter(
+    (y) => y !== collection[collection.findIndex(
+      (x) => x.id === parseInt(buttonId, 10),
```

Comment:
- [ ] The issue with the positioning of `id`'s of entries highlighted in the previous review still persists. I kindly entreat you to spend some time to carefully resolve this issue. 

**Sample of locally stored data in your app:**
![image](https://user-images.githubusercontent.com/12745474/130068761-182f402f-56c6-46ed-b2d5-c4d8fea455c7.png)